styles.css

```diff
@@ -0,0 +1,26 @@
+form {
+  width: 30%;
+  height: 30%;
+  padding-left: 80px;
+  padding-right: 80px;
+  padding-top: 80px;
+  padding-bottom: 40px;
+  margin: 0 auto;
+  margin-top: 150px
+}
+
+.hidden {
+  visibility: hidden
+}
+
+#question {
```

Comment:
Try using bootstrap's `form-group` class and let's see what the margin looks like.